Documentation/Troubleshooting/ceph-csi-common-issues.md

```diff
@@ -425,40 +425,42 @@ kubectl -n rook-ceph delete pod my-app-69cd495f9b-nl6hf --grace-period 0 --force
 
 After the force delete, wait for a timeout of about 8-10 minutes. If the pod still not in the running state, continue with the next section to blocklist the node.
 
```

Comment:
How about a new section on how to get the IP of the holder pod?

Reply:
added a sub section to get the holder Pod IP.